<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNAWriteDialog</class>
 <widget class="QDialog" name="QNAWriteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="btn_sendcansel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_QnA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>QnA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>289</width>
      <height>179</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="textEdit_Write">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
